acceptance/repository-postgresql/test/schema.sql

```diff
@@ -0,0 +1,8 @@
+-- Seed database schema & data
+-- See https://github.com/strongloop/loopback-connector-postgresql/blob/master/test/schema.sql
+
+DROP DATABASE IF EXISTS repository_tests;
+CREATE DATABASE repository_tests;
+
+DROP DATABASE IF EXISTS repository_tests_new;
```

Comment:
Do we still need the second database if we can have `repository_tests` created?

Reply:
Yeah, we still need to manually create it for [this test]( https://github.com/strongloop/loopback-next/blob/postgresql-tests/packages/repository-tests/src/crud/transactions.suite.ts#L144-L149).

Reply:
> > Regarding
> > > Re the coverage drop: the transactions test suite isn't run by the memory database, but it's run by acceptance/mysql and acceptance/postgresql.
> > 
> > 
> > Do you have a proposed fix (new set of tests) to bring the coverage back up to an appropriate level?
> > Right now, the CI is complaining
> > ![image](https://user-images.githubusercontent.com/6864736/67041146-069a3780-f0f3-11e9-8f5a-c65a2217545c.png)
> 
> @emonddr Not really sure how to bring it back up because as I said the coverage only accounts for tests run by the memory database and the memory database doesn't support transactions. So the coverage for the [file](https://github.com/strongloop/loopback-next/pull/3853/files#diff-5b8b9c9868638cbf9ea2dfbbee7b1593) it's complaining about can't be brought back up. But do you have something in mind?

@bajtos @nabdelgadir , should we use

https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md

statements for the code block you are mentioning in the file link above?

Or is there a better solution?

Reply:
Thanks, I updated to use this!